commit/merkleroot/rmn/types.go

```diff
@@ -0,0 +1,36 @@
+package rmn
+
+import "github.com/smartcontractkit/chainlink-common/pkg/types/ccipocr3"
+
+type NodeSignatures struct {
+	// Signatures are the ECDSA signatures for the lane updates for each node.
+	// NOTE: A signature[i] corresponds to the whole LaneUpdates slice and NOT LaneUpdates[i].
+	Signatures  []ECDSASignature
+	LaneUpdates []FixedDestLaneUpdate
+}
+
+type ECDSASignature struct {
+	R []byte
+	S []byte
+}
```

Comment:
Please use a stdlib, perhaps [crypto/ecdsa](https://pkg.go.dev/crypto/ecdsa)?

Reply:
I guess if they're opaque types it doesn't really matter. They just get passed along to the contracts or will we verify them offchain?

Reply:
We can verify them offchain for extra safety.

Reply:
Yeah we will have an offchain version of the RMNRemote.verify function (here: https://github.com/smartcontractkit/chainlink/blob/70223b96e9862c02e6779cf37cc5d358766b0ea2/contracts/src/v0.8/ccip/rmn/RMNRemote.sol#L111-L123)